app/services/jobs/notifications/notifications-status-updates.service.js

```diff
@@ -5,10 +5,44 @@
  * @module ProcessNotificationsStatusUpdatesService
  */
 
+const FetchService = require('./fetch-event-notifications.service.js')
+const NotifyStatusPresenter = require('../../../presenters/notifications/setup/notify-status.presenter.js')
+const NotifyStatusService = require('../../notify/notify-status.service.js')
+const UpdateNotificationsService = require('../../notifications/setup/update-notifications.service.js')
+const { timestampForPostgres } = require('../../../lib/general.lib.js')
+
 /**
  * Updates the statuses of notifications
  */
-async function go() {}
+async function go() {
+  const events = await FetchService.go()
+
+  for (const event of events) {
+    await _processEvent(event)
+  }
+}
+
+async function _processEvent(event) {
+  const notifications = []
+
+  for (const scheduledNotification of event.scheduledNotifications) {
+    const notifyResponse = await NotifyStatusService.go(scheduledNotification.notifyId)
+
+    if (!notifyResponse.errors) {
```

Comment:
How should we handle the error. Shall we log each error as it comes. 

It will be 4xx / 5xx.